client/node-common/src/service/node_builder.rs

```diff
@@ -62,6 +64,10 @@ use {
 
 #[allow(deprecated)]
 use sc_executor::NativeElseWasmExecutor;
+use sc_network::request_responses::IncomingRequest;
```

Comment:
We have a lot of places with 2 different `use` styles - combined and separate lines.
Should we follow one of them? Or should we use different ones depending on specific conditions? Like here - you have aggregated `use` and you add separated lines?